2024-11-26 14:18:39,687 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-26 14:18:39,687 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Configure stats pid to 197009
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Loading settings from /home/danfossautonomypc/.config/wandb/settings
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Loading settings from /home/danfossautonomypc/segmentation/wandb_segment/wandb/settings
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': 'online', '_disable_service': None}
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'try.py', 'program_abspath': '/home/danfossautonomypc/segmentation/wandb_segment/try.py', 'program': '/home/danfossautonomypc/segmentation/wandb_segment/try.py'}
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_init.py:_log_setup():533] Logging user logs to /home/danfossautonomypc/segmentation/wandb_segment/wandb/run-20241126_141839-use_remote_data/logs/debug.log
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_init.py:_log_setup():534] Logging internal logs to /home/danfossautonomypc/segmentation/wandb_segment/wandb/run-20241126_141839-use_remote_data/logs/debug-internal.log
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_init.py:init():619] calling init triggers
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_init.py:init():669] starting backend
2024-11-26 14:18:39,688 INFO    MainThread:197009 [wandb_init.py:init():673] sending inform_init request
2024-11-26 14:18:39,690 INFO    MainThread:197009 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: fork
2024-11-26 14:18:39,690 INFO    MainThread:197009 [wandb_init.py:init():686] backend started and connected
2024-11-26 14:18:39,693 INFO    MainThread:197009 [wandb_init.py:init():781] updated telemetry
2024-11-26 14:18:39,694 INFO    MainThread:197009 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-26 14:18:56,577 WARNING MainThread:197009 [wandb_init.py:init():1260] interrupted
Traceback (most recent call last):
  File "/home/danfossautonomypc/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
  File "/home/danfossautonomypc/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 817, in init
    result = run_init_handle.wait(
  File "/home/danfossautonomypc/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 279, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/danfossautonomypc/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/danfossautonomypc/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 122, in _wait
    return self._event.wait(timeout=timeout)
  File "/usr/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-11-26 14:18:56,583 WARNING MsgRouterThr:197009 [router.py:message_loop():75] message_loop has been closed
